### DISCUSSION TITLE
How to make output bounding boxes thinner ?

### DISCUSSION THREAD

**Discussion by Samuel-Bachorik** [ultralytics/yolov5#2444] [General]
Hi ! Now for me Yolo is working perfectly but I have one question about output boxes. When I use low resolution of video these boxes are very Thick. I want them to be thinner on low resolution images. Can you please show me what should I adjust ? 
Photo what I mean - https://ibb.co/bFMSXXS Text and boxes are very Thick and big 

Thank you

**Comment by kinoute** [comment]
You can try to change the `line_thickness` parameter of the `plot_one_box()` function here:

https://github.com/ultralytics/yolov5/blob/886f1c03d839575afecb059accf74296fad395b6/utils/plots.py#L57

The default is set to 3, 1 might be better.

**Reply by glenn-jocher** [reply]
We probably want a better linewidth and fontsize scaling method that works across a wider range of sizes (i.e. 128-4k). We attempt some autoscaling now (L59, L65) but it mostly looks best at 640-1280.
https://github.com/ultralytics/yolov5/blob/886f1c03d839575afecb059accf74296fad395b6/utils/plots.py#L57-L69
